### PR TITLE
BD-3664 Outgrow partner docs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Why are you making this change? (required)

Add Outgrow docs

### Related PRs, issues, or features (optional)

[BD-3664](https://jira.braze.com/browse/BD-3664)

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by internetisaiah — CHANGES_REQUESTED** [review]
@rachel-feinberg , heads up: you'll need to add the link for `outgrow.md` to all the landing pages of the parent directories, up till `_partners`. so there should be about 3 landing pages modified in total (and you already did 1). 👍🏽

**Review by internetisaiah — APPROVED** [review]
LGTM, feel free to merge whenever the partner is ready to go live 👍🏽

File: _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+4. Customize your campaign content and settings, and then launch.
+
+## Troubleshooting common issues
+
+| Issue | Solution |
+|-------|----------|
+| **Data isn't transferring to Braze** | Verify that the API key and endpoint URL are correct in your Outgrow integration settings. Make sure the API key has the `users.track` permission turned on. |
+| **Incorrect data mapping** | Make sure that each mapped Outgrow response corresponds to a valid Braze custom attribute and that the attribute names match exactly. |
+| **Segment not filtering correctly** | Make sure that custom attributes in Braze are properly set up and receiving data. Re-check your segment filter logic. |
+{: .reset-td-br-1 .reset-td-br-2 role="presentation" }git
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md:100]:
```suggestion
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }
```
I think this is breaking your table

File: _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Prerequisites
+
+Before setting up the Outgrow and Braze integration, confirm you have the following:
+
+| Requirement | Description |
+|-------------|-------------|
+| **Outgrow account** | A registered Outgrow account to configure and manage interactive content and data transfer settings |
+| **Braze account** | A Braze account with access to REST API credentials |
+| **API key** | An API Key from Braze with appropriate permissions (such as `users.track`) to enable user data transfer. |
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md:30]: I wonder if we should be crisper about what the "appropriate permissions" are? If we don't know, how would the user?

Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md:30]:
```suggestion
| **API key** | An API Key from Braze with appropriate permissions (such as `users.track`) to enable user data transfer |
```

If the other table entries don't have a period, let's remove it here, too.

File: _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+   - Filter by `outgrow_quiz_score` to target users who scored above a certain threshold.
+   - Filter by `outgrow_custom_segment` to target users who belong to a particular Outgrow-defined segment.
+3. Save your segment for use in campaigns and Canvases.
+
+### Launching campaigns with Outgrow-defined segments
+
+You can use the custom segments created from Outgrow data to personalize your Braze campaigns and target users based on their responses to interactive content. To do so and create a more personalized user experience, follow these steps:
+
+1. In Braze, go to **Engagement** > **Campaigns**.
+2. Select **Create Campaign** and choose your campaign type (email, push, in-app message, or others.).
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/data_and_infrastructure_agility/leads_capture/outgrow.md:89]:
```suggestion
2. Select **Create Campaign** and choose your campaign type (email, push, in-app message, or others).
```

**Review by josh-mccrowell-braze — APPROVED** [review]
A few nitpicks, but otherwise looking good to go!

**Pull request merged**